packages/botonic-plugin-contentful/src/contentful/delivery-api.ts

```diff
@@ -200,3 +141,18 @@ export class ContentfulEntryUtils {
     })
   }
 }
+
+export function createContentfulClientApi(
```

Comment:
Do we need here the "cache" variable? Or this new "Client" ignores the cache?

Reply:
this is the lowest level client to contentful, without the cache.